### PR TITLE
Fix duplicate playlists once playlist is confirmed  / Await feature flag response in sagas C-1199

### PR DESCRIPTION
**Pull request by nicoback2**
### Description
Problems were:
1. Not awaiting the `getFeatureFlag` call in sagas
2. Not filtering out _moved playlists

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/common/src/store/pages/profile/selectors.ts [hunk 1]
```diff
@@ -75,7 +75,7 @@ export const getProfileCollections = createDeepEqualSelector(
       .filter((collection) => {
         if (collection) {
           const { is_delete, _marked_deleted, _moved } = collection
-          return !(is_delete || _marked_deleted) || _moved
+          return !(is_delete || _marked_deleted || _moved)
```
Comment (nicoback2) [inline packages/common/src/store/pages/profile/selectors.ts:78]: @dylanjeffers can you double check this? I'm not sure if it was supposed to be `|| _moved` but that seemed incorrect?

File: packages/web/src/common/store/cache/collections/sagas.js [hunk 2]
```diff
@@ -77,8 +77,10 @@ function* createPlaylistAsync(action) {
   const userId = yield select(getUserId)
   const uid = action.playlistId
   const getFeatureEnabled = yield getContext('getFeatureEnabled')
-  const playlistEntityManagerIsEnabled =
-    getFeatureEnabled(FeatureFlags.PLAYLIST_ENTITY_MANAGER_ENABLED) ?? false
```
Comment (nicoback2) [inline packages/web/src/common/store/cache/collections/sagas.js:81]: Previously we weren't awaiting `getFeatureEnabled` so this was just returning a Promise which is truthy. So `playlistEntityManagerIsEnabled` was always true in this file :O

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--dupe-playlist-fix

File: packages/web/src/common/store/cache/collections/sagas.js (hunk 2 above)
Comment (isaacsolo) [reply packages/web/src/common/store/cache/collections/sagas.js:81]: omg

**Review by isaacsolo — APPROVED** [review]

**Review by sliptype — APPROVED** [review]
Great finds

**Pull request merged**